server/aws/src/.env.example

```
@@ -2,20 +2,26 @@
+NONCES_TABLE_NAME=ResourcesStack-DataStoresNoncesTableXXXXXXXX-XXXXXXXXXXXX
+# Required for DynamoDB pre-authorized code store (deploy output: PreCodesTableName)
+PRE_CODES_TABLE_NAME=ResourcesStack-DataStoresPreCodesTableXXXXXXXX-XXXXXXXXXXXX
 
 # Authz (default: start:authz)
 # AUTHZ_PORT=8082
 # AUTHZ_BASE_URL=http://localhost:8082
 # Required for DynamoDB authz server metadata store (deploy output: AuthServersTableName)
 AUTH_SERVERS_TABLE_NAME=ResourcesStack-DataStoresAuthServersTableXXXXXXXX-XXXXXXXXXXXX
-# PRE_CODES_TABLE_NAME=ResourcesStack-...
+# Required for DynamoDB pre-authorized code store (deploy output: PreCodesTableName)
+PRE_CODES_TABLE_NAME=ResourcesStack-DataStoresPreCodesTableXXXXXXXX-XXXXXXXXXXXX
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The PRE_CODES_TABLE_NAME key is duplicated

(DuplicatedKey)